doc/database/README.md

```diff
@@ -0,0 +1,65 @@
+# Database
+
+This README.md file provides a comprehensive documentation of the database, including its structure, guidelines, and conventions.
+The purpose of this documentation is to ensure consistency, readability, and ease of use for all users and developers interacting with the database.
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Database Schema](#database-schema)
+3. [Entity-Relationship Diagram](#entity-relationship-diagram)
+4. [Table Descriptions](#table-descriptions)
+5. [Database Guidelines](#database-guidelines)
+6. [Contributing](#contributing)
+
+## Introduction
+
+This database is designed to store and manage information related to plants, their properties, cultivation details, environmental preferences, and more.
+It follows a set of guidelines and conventions to ensure consistency and maintainability.
+
+## Database Schema
+
+The database schema consists of multiple tables and relationships, including:
+
+- `plants`
+- `seeds`
+- `species`
+- `genera`
+- `subfamilies`
+- `families`
+
+Enums are used to represent predefined sets of values, such as:
+
+- `tag`
+- `quantity`
+- `quality`
+
+For further information, see [Database Schemata](./database_schemata.md).
+
+## Entity-Relationship Diagram
+
+The ER diagram provides a visual representation of the tables, relationships, and cardinalities within the database.
+This diagram can be created using Mermaid syntax and then viewed using a Mermaid-compatible viewer.
```

Comment:
mdBook should be able to render it (@kitzbergerg correct me if I am wrong)

Reply:
I just looked it up right now and there is a mdbook plug-in called mdbook-fishextract which can render mermaid syntax

Reply:
I just added mermaid as part of #307. So as soon as that is merged we should be able to see if it works.